x-pack/plugins/lens/public/drag_drop/providers.tsx

```diff
@@ -176,7 +258,12 @@ export function ReorderProvider({
             </p>
             <p id={`lnsDragDrop-reorderInstructions-${id}`}>
```

Comment:
Looks like this generated `lnsDragDrop-reorderInstructions-x` which is never referenced by any button. (Same below.)

Reply:
Let's talk offline about it!